src/components/List/Card/Card.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from '../List.module.scss';
+import { Book } from '../../../types/main';
+
+class Card extends React.PureComponent<{ card: Book }> {

```

Comment:
```suggestion
class Card extends React.Component<{ card: Book }> {
```
No state is used. So no need to use PureComponent.